### PR TITLE
Bugs alignments and fleed correction

### PR DESCRIPTION
**Pull request by Landos007**

### CODE REVIEW

File: src/components/sections/information.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { Link } from "gatsby"
```
Comment (musayann) [inline src/components/sections/information.js]: I thought this page had to be removed

File: src/pages/page-2.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { Link } from "gatsby"
+
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+const SecondPage = () => (
```
Comment (musayann) [inline src/pages/page-2.js]: Please delete this page

**Review by musayann — CHANGES_REQUESTED** [review]

**Comment by musayann** [comment]
@Landos007 Please capitalize the PR's title and provide a description for the PR

File: src/components/sections/information.js (hunk 1 above)
Comment (Landos007) [reply src/components/sections/information.js:1]: which one?

Comment (Landos007) [reply src/components/sections/information.js:1]: the member.js?

Comment (Landos007) [reply src/components/sections/information.js:1]: or information.js

**Review by musayann — APPROVED** [review]

**Pull request merged**